shared-module/board/__init__.c

```diff
@@ -179,4 +183,7 @@ void reset_board_busses(void) {
 #if BOARD_UART
     MP_STATE_VM(shared_uart_bus) = NULL;
 #endif
+#if CIRCUITPY_RP2PIO
+    reset_rp2pio_statemachine();
```

Comment:
I don't think we need this anymore because `port_reset` should call `reset_rp2pio_statemachine`. `rp2pio` is meant to be port-specific so it shouldn't influence any code outside of ports/raspberrypi

Reply:
Removed it and still tested OK. Thanks!